image/roxctl/Dockerfile.rhtap

```diff
@@ -0,0 +1,53 @@
+# TODO: can we follow tags?
```

Comment:
Regarding filename. I learned that some IDEs (GoLand?) consider Dockerfile to be an extension, not a file name, for syntax highlighting to work. Therefore, I suggest renaming `Dockerfile.rhtap` -> `rhtap.Dockerfile`, if that still works.

Reply:
Yes that will work. The path to the Dockerfile within the repository is fully configurable within the Tekton pipeline. I added a separate file to keep building roxctl image with GHA for now.